### PR TITLE
Update orgs preview page with org_api and orgs-cli

### PR DESCRIPTION
**Pull request by bgrenon**
Private preview page:
https://neon-next-git-bgrenon-org-api-private-preview-neondatabase.vercel.app/docs/manage/organizations#managing-projects-using-the-neon-api

CLI page:
https://neon-next-git-bgrenon-org-api-private-preview-neondatabase.vercel.app/docs/reference/cli-orgs
https://neon-next-git-bgrenon-org-api-private-preview-neondatabase.vercel.app/docs/reference/neon-cli

### CODE REVIEW

**Review by danieltprice — APPROVED** [review]

File: content/docs/manage/organizations.md [hunk 1]
```diff
@@ -165,13 +165,262 @@ As the Admin for the organization account:
...
+
+You can also use the `org_id` parameter in `GET` requests to:
+
+- List all projects that belong to your organization
+- Get consumption metrics for your organization
+- List of all organizations that you belong to (via your personal account)
+
+### Finding your org_id
+
+While you can't get your `org_id` directly in the Neon Console (it's not yet available during private preview), if you navigate to your organization, you can find the ID in the console URL:
```
Comment (adi-griever) [inline content/docs/manage/organizations.md:182]: maybe worth mentioning org_id always starts with "org-". Might be easier for customers to understand what it is

Comment (bgrenon) [reply content/docs/manage/organizations.md:182]: Good point. Added.

File: content/docs/manage/organizations.md [hunk 2]
```diff
@@ -165,13 +165,284 @@ As the Admin for the organization account:
...
+
+![org ID in console URL](/docs/manage/org_id_URL.png)
+
+<Admonition type="tip">
+To help you recognize organization IDs, note that they all start with the prefix `org`.
```
Comment (danieltprice) [inline content/docs/manage/organizations.md:187]: take or leave:

To help you identify organization IDs more easily, remember that they all begin with the prefix "org".

File: content/docs/manage/organizations.md [hunk 3]
```diff
@@ -165,13 +165,284 @@ As the Admin for the organization account:
...
+      "updated_at": "2022-12-04T02:39:25Z"
+    },
+    ...
+  ]
+}
+```
+
+## Managing organizations using the Neon CLI
+
+During private preview, we'll be continuing to add options for managing your organization using the Neon CLI. For now, we've added the main command `neonctl orgs` with a single subcommands `list`, which outputs a list of all organizations that the CLI user currently belongs to.
```
Comment (danieltprice) [inline content/docs/manage/organizations.md:423]:
```suggestion
During private preview, we'll be continuing to add options for managing your organization using the Neon CLI. For now, we've added the main command `neonctl orgs` with a single subcommand `list`, which outputs a list of all organizations that the CLI user currently belongs to.
```

**Review by danieltprice — APPROVED** [review]

File: content/docs/manage/organizations.md (hunk 2 above)
Comment (bgrenon) [inline content/docs/manage/organizations.md:187]:
```suggestion
To help you identify organization IDs more easily, notice that they all start with the prefix `org`.
```

**Pull request merged**